rpms/generate_rpm_package.sh

```diff
@@ -258,6 +266,7 @@ main() {
         build || exit 1
     fi
 
+
```

Comment:
This line is unnecessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/9fdbf7e2f0271d39ecdb594e355a87b08d902054.